omnibus/partybus/lib/partybus.rb

```diff
@@ -21,6 +21,16 @@ class Config
     attr_accessor :bootstrap_server
     attr_accessor :couchdb_data_dir
 
+    attr_accessor :running_server
+    attr_accessor :postgres
+
+    def initialize
+      if File.exists?("/etc/opscode/chef-server-running.json")
```

Comment:
Should we be throwing a big error here if chef-server-running.json doesn't exist?